Formula/aztfy.rb

```diff
@@ -0,0 +1,23 @@
+class Aztfy < Formula
+  desc "Bring your existing Azure resources under the management of Terraform"
+  homepage "https://azure.github.io/aztfy"
+  url "https://github.com/Azure/aztfy.git",
+      tag:      "v0.5.0",
+      revision: "f957fc158684aaf66551543327124865c7fe4eca"
+  license "MPL-2.0"
+
```

Comment:
add head build as well
```suggestion
  license "MPL-2.0"
  head "https://github.com/Azure/aztfy.git", branch: "main"
```

Reply:
I appreciate your help!! 😄